packages/composables/src/useMedia/useMedia.ts

```diff
@@ -0,0 +1,35 @@
+import { useShopwareContext } from "#imports";
+import type { operations } from "#shopware";
+
+interface UseMedia {
```

Comment:
it's our first `interface` within composables, maybe we could keep it consistent and change to the type 😉